src/apps/AppCenter/AppCenter.js

```diff
@@ -134,83 +122,55 @@ class AppCenter extends React.Component {
+      compactMode,
+      reposLoading,
+      onUpgradeAll,
+      canUpgradeOrg,
+    } = this.props
     const { upgradePanelOpened } = this.state
     const { activeTab, openedRepoName } = this.getLocation()
     const repos = this.getRepos()
     const currentRepo = openedRepoName && this.getRepoFromName(openedRepoName)
 
-    const navigationItems = [
-      'App Center',
-      ...(currentRepo ? [currentRepo.name] : []),
-    ]
-
     return (
       <React.Fragment>
-        <AppView
-          style={{ height: '100%' }}
-          appBar={
-            <Viewport>
-              {({ below, above }) => (
-                <AppBar
-                  tabs={
-                    openedRepoName ? null : (
-                      <div
-                        css={`
-                          margin-left: ${below('medium') ? '-14px' : '0'};
-                        `}
-                      >
-                        <TabBar
-                          items={SCREENS.map(screen => screen.label)}
-                          selected={activeTab}
-                          onChange={this.handleScreenChange}
-                        />
-                      </div>
-                    )
-                  }
-                >
-                  {below('medium') && navigationItems.length < 2 && (
-                    <MenuButton
-                      onClick={this.handleMenuPanelOpen}
-                      css={`
-                        position: relative;
-                        z-index: 2;
-                        margin-left: 0;
-                        margin-right: -24px;
-                      `}
-                    />
-                  )}
-                  <NavigationBar
-                    items={navigationItems}
-                    onBack={this.handleCloseRepo}
-                  />
-                  {canUpgradeOrg && !openedRepoName && (
-                    <UpgradeButton
-                      mode={below('medium') ? 'text' : 'strong'}
-                      onClick={onUpgradeAll}
-                      title="Upgrade all"
-                    >
-                      {below('medium') ? <IconUpgrade /> : 'Upgrade all'}
-                    </UpgradeButton>
-                  )}
-                </AppBar>
-              )}
-            </Viewport>
-          }
-        >
-          {activeTab === 0 &&
-            (reposLoading ? (
-              <EmptyBlock>Loading apps…</EmptyBlock>
-            ) : (
-              <InstalledApps
-                onOpenApp={this.handleOpenRepo}
-                onRequestUpgrade={this.handleOpenUpgradePanel}
-                openedRepoId={currentRepo && currentRepo.appId}
-                repos={repos}
+        <Header
+          primary="App center"
+          secondary={
+            canUpgradeOrg &&
+            activeTab !== 1 &&
+            !openedRepoName && (
+              <Button
+                mode="strong"
+                onClick={onUpgradeAll}
+                label="Upgrade all"
```

Comment:
cc @dizzypaty Perhaps we can change this to "Upgrade to 0.8" since it's not really "upgrade all" (it won't upgrade apps like TPS).